assets/scss/6-components/series-promo/_series-promo.scss

```diff
@@ -0,0 +1,111 @@
+// Series promo (c-series-promo)
+//
+// This is a one-off feature used on the [series page for "In Harvey's Wake"](https://www.texastribune.org/series/in-harveys-wake/). We could potentially repeat this pattern in the future, but may want to consider custom-properies for new color schemes.
+//
+// Keywords: flatblock
+//
+//
+// Markup: 6-components/series-promo/series-promo.html
+//
+// Styleguide 6.1.5
+.c-series-promo {
+  background-size: cover;
+  background-position-x: center;
+  background-position-y: top;
+
+  &__title {
+    color: $color-sand;
+  }
+
+  &__button {
+    background-color: $color-sand;
+    color: $color-blue-dark;
+    display: block;
+    // non-standard letter-spacing setting
+    letter-spacing: .1rem;
+  }
+
+  &__list {
+    color: $color-sand;
+    font-size: $size-m;
+
+    li {
+      margin-bottom: $size-b;
+      margin-left: $size-l;
+    }
+
+    li:last-child {
+      margin-bottom: 0;
+      margin-left: 0;
+      list-style: none;
+    }
+  }
+  @include mq($from: bp-s) {
+    height: 350px;
+    display: grid;
+
+    &__title {
+      color: $color-blue-dark;
+    }
+
+    &__button {
+      background-color: $color-blue-dark;
+      color: $color-sand;
+    }
+
+    &__list {
+      background-color: $color-sand;
+      color: $color-blue-dark;
+    }
+
+    &--harvey-1 {
+      background-image: url('https://static.texastribune.org/media/images/2017/10/16/TT-siderail-ad-harvey-walking-bg.JPG');
+      background-position-y: 45%;
+      background-position-x: 55%;
+    }
+
+    &--harvey-2 {
+      background-image: url('https://static.texastribune.org/media/images/2017/10/16/TT-siderail-ad-harvey-neighborhood-bg.jpg');
+    }
+  }
```

Comment:
Referencing the background images like this feels a little brittle (if one accidentally gets deleted from our file system). I wonder if it's possible to include the images inside this repo, since presumably any time you import this partial you're also gonna need the art. That said, background-image paths in CSS/SCSS have always thrown me for a loop, so maybe what I'm asking is weird.

Reply:
That is a good point. I struggled with how "real" to make these and if I should abstract them out more. I concluded that I'd need to see if we even reuse this one again to know how to generalize it. I agree that we should stick to hosting the asset here, just to keep it all living together. I'll make that update. Thanks!